pkgs/development/ocaml-modules/capnp/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, base_quickcheck
```

```suggestion
, base_quickcheck ? null
```

Technical detail to fix evaluation.

---

Sounds good -- done!